contracts/src/ModuleCRC21.sol

```diff
@@ -8,9 +8,9 @@ contract ModuleCRC21 is DSToken {
     string denom;
     bool isSource;
 
-    event __CronosSendToIbc(address sender, string recipient, uint256 amount);
-    event __CronosSendToChain(address sender, address recipient, uint256 amount, uint256 bridge_fee, uint256 chain_id);
-    event __CronosCancelSendToChain(address sender, uint256 id);
+    event __CronosSendToIbc(address indexed sender, string indexed recipient, uint256 amount, string indexed channel_id, bytes extraData);
```

Comment:
I think it'd be clearer to change the name as well, sth like `__CronosSendToIbcV2`?

Reply:
params are lengthy here, not sure if need make use of [struct](https://solidity-by-example.org/structs/) with higher solidity

Reply:
not a strong opinion but since previous `__CronosSendToIbc` will be deprecated (still supported but should not be used) not sure if there is any reason to carry a v2.

there is little utilisation of this event atm also I think